IyiKuyoro/Pals
# #161723921 update README

### What does this PR do?
Updates the README and fixes issue with the TravisCI badge.

### How can this be manually tested
> - Visit the branch on the GitHub repo
> - Check the README and click the Travis badge
> - You should be redirected to the develop branch build process on Travis

### Relevant PT story
#161723921
